docs/book/intro.md

```diff
@@ -129,8 +129,7 @@ the default configuration and your [PSR-11 container](https://www.php-fig.org/ps
 cannot be detected automatically, you can still use the laminas-cli tool.
 
 To integrate such applications with laminas-cli, you will need to create a file
-that returns a PSR-11 container. Do so in the file `config/container.php`. Such
-a file might look like the following:
+that returns a PSR-11 container. Do so in the file `config/container.php` or if you already have such a file in another location, use the `--container=<path>` option. Such a file might look like the following:
```

Comment:
```suggestion
that returns a PSR-11 container.
Do so in the file `config/container.php` or if you already have such a file in another location, use the `--container=<path>` option.
Such a file might look like the following:
```

(new style guide suggests one sentence per line, so I've wrapped the new sentences to their own lines here)